backupzip/backupzip.go

```diff
@@ -0,0 +1,61 @@
+package backupzip
+
+import (
+	"archive/zip"
+	"io"
+	"os"
+	"path/filepath"
+)
+
```

Comment:
Let's try and get into the habit of commenting our public functions with descriptions of what they do.